README.md

```diff
@@ -103,4 +106,27 @@ Space: O(1)
 - Travis Cox
 
 #### Solution
-N/A
\ No newline at end of file
+N/A
+
+--------------------------------------------------------------
+
+### 5. Linked list insertions
+
+#### Challenge
+Write 3 insertion methods for a singly linked list.
+
+#### Approach & Efficiency
+Continue working off of the already built LinkedList & Node class.  Implement 3 insertion methods to append a node at the end, insert before a node, and insert after a node.
+
+Big O Notation
```

Comment:
For this code challenge the methods all happen to have to same time and space complexity. Moving forward you should be sure to specify Big O for each method you define